gram5/GRAM5_Fixed_Bugs_Frag.adoc

```diff
@@ -2,11 +2,7 @@
 [[gram5-fixed-bugs]]
 === Fixed Bugs for GRAM5 ===
 
-
```

Comment:
How should this be changed instead? And how do we know these fixed bugs were fixed with GT6.0 and not GT5.2 or so? I don't want to cross-check this when we'll have a copy of 4afb016bf34876b07a112cb15cf70d5683f70f13 as GCT/GT6.0 documentation in the future adjacent to the GCT6.2 documentation.